angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html

```diff
@@ -14,18 +14,13 @@
         <input type="text" class="visible-xs marketplace-search" placeholder="What are you looking for?" ng-model="searchText" select-on-page-load aria-label="Search bar enter the app you are looking for">
       </div>
     </div>
-    <div class="mp-opaque">
-      <ul>
-        <li ng-click="selectFilter('popular','')" ng-class="{true:'selected-filter'}[selectedFilter === 'popular']"><a href="#" aria-label="search from the most popular apps">Most Popular</a></li>
-        <li ng-click="selectFilter('az','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'az']"><a href="#" aria-label="search by alphabetical order">A-Z</a></li>
-        <li ng-click="selectFilter('category','')" ng-class="{true: 'selected-filter'}[selectedFilter === 'category']"><a href="#" aria-label="search by categories">Categories</a></li>
+    <div class="inner-nav-container">
+      <ul class="inner-nav">
+        <li ng-click="selectFilter('popular','')" ng-class="{true:'active'}[selectedFilter === 'popular']"><a href="#" aria-label="search from the most popular apps">Most Popular</a></li>
```

Comment:
Not a problem introduced by this changeset, but: the `aria-label`s here seem goofy. This is more "Sort by popularity", "Sort alphabetically", "Filter by categories", say. "search by alphabetical order" seems an odd way to say that.